internal/wallet/wallet.go

```diff
@@ -28,89 +19,7 @@ import (
 
 var SecretBytes = hashing.New([]byte("aurum"))[8:16]
```

Comment:
Do we use this anymore anywhere?

Reply:
It is used in `RequestWalletInfo`.

Reply:
That's my mistake. I don't think we need that function anymore since we're using `http`